src/http_crawler/__init__.py

```diff
@@ -19,7 +19,10 @@ def crawl(base_url, follow_external_links=True):
 
     while todo:
         url = todo.pop()
-        rsp = session.get(url)
+        rsp = session.get(url, allow_redirects=follow_redirects)
+
+        if rsp.status_code // 100 == 3:
+            continue
```

Comment:
I think we should still yield a response, even if it is a redirect.  For instance, a user of the library might want to know about all requests that are redirected.  We should also extract any URLs from the Location header of a 3xx response.

Please could you:

 * remove these two lines, so that we continue to yield all responses;
 * add a check [in this section](https://github.com/inglesp/http-crawler/blob/master/src/http_crawler/__init__.py#L31-L37) so that if the response is a 3xx, we try to extract a URL from the Location header.

Does that make sense?

Once you've done that, you should also add yourself to [AUTHORS.txt](https://github.com/inglesp/http-crawler/blob/master/AUTHORS.txt)